src/manifest/mod.ts

```diff
@@ -50,6 +50,7 @@ export class SlackManifest {
       oauth_config: {
         scopes: {
           bot: this.ensureBotScopes(),
+          user: def.userScopes,
```

Comment:
How does the API respond to user scopes being set to an empty array?

Reply:
Tried to force a token_management_enabled=true + oauthconfig.scopes.user = [] scenario to see what the backend says, but it seems like today due to the way the go marshals empty arrays (omitempty), it gets removed entirely from the payload sent to Slack. Kind of surprising behavior, since i guess we could technically pass this kind of test at the SDK level, but it wouldn't be reflective of what the CLI sends over to Slack.

```

  const definition2: SlackManifestType = {
    runOnSlack: true,
    name: "fear and loathing in las vegas",
    description:
      "fear and loathing in las vegas: a savage journey to the heart of the american dream",
    displayName: "fear and loathing",
    icon: "icon.png",
    botScopes: ["channels:history", "chat:write", "commands"],
    userScopes: [], // empty user scopes
  };
  const manifest2 = Manifest(definition2);

  assertStrictEquals(
    manifest2.oauth_config.scopes.user,
    definition2.userScopes,
  );
```